src/libos/src/fs/dev_fs/dev_random.rs

```diff
@@ -14,14 +14,6 @@ impl INode for DevRandom {
         Err(FsError::PermError)
     }
 
-    fn poll(&self) -> vfs::Result<vfs::PollStatus> {
```

Comment:
For dev/random, the returned `PollStatus::write`should be false.

Reply:
Ok, the `poll` of inode in devfs may be special.